text/0026-rest-api/rest-api.yaml

```diff
@@ -0,0 +1,1442 @@
+              type: string
+            maxResults:
+              type: integer
+            count:
+              type: integer
+            messageIds:
+              type: array
+              items:
+                type: string
+    GetMessageMetadataByIdResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            messageId:
+              type: string
+            parent1MessageId:
+              type: string
+            parent2MessageId:
+              type: string
+            isSolid:
+              type: boolean
+            referencedByMilestoneIndex:
+              type: integer
+              nullable: true
+            milestoneIndex:
+              type: integer
+            ledgerInclusionState:
+              enum:
+                - included
+                - conflicting
+            conflictReason:
+              type: integer
+            shouldPromote:
+              type: boolean
+            shouldReattach:
+              type: boolean
+          required:
+            - messageId
+            - parent1MessageId
+            - parent2MessageId
+            - isSolid
+    GetMessageByIdDataResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            allOf:
+              $ref: '#/components/schemas/Message'
+    GetMessageChildrenResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            messageId:
+              type: string
+            maxResults:
+              type: integer
+            count:
+              type: integer
+            childrenMessageIds:
+              type: array
+              items:
+                type: string
+    GetOutputsByIdResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            messageId:
+              type: string
+            transactionId:
+              type: string
+            outputIndex:
+              type: integer
+            isSpent:
+              type: boolean
+            output:
+              anyOf:
+                - $ref: '#/components/schemas/SigLockedSingleOutput'
+    GetAddressBalanceResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            address:
+              type: string
+            maxResults:
+              type: integer
+            count:
+              type: integer
+            balance:
+              type: integer
+    GetAddressOutputsResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            address:
+              type: string
+            maxResults:
+              type: integer
+            count:
+              type: integer
+            outputIDs:
+              type: array
+              items:
+                type: string
+    GetMilestoneByIndexResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            milestoneIndex:
```

Comment:
As we changed this to `index` in the event channels, we should change it here too.